pkg/rehearse/jobs.go

```diff
@@ -576,6 +587,7 @@ func selectJobsForRegistryStep(node registry.Node, configs []*config.DataWithInf
 			}
 		}
 	}
+	loggers.Debug.WithField("node-name", node.Name()).Debug("Found no jobs using node")
```

Comment:
Yeah this would really benefit from the logrus hooks impl, no reason to ask the devs to choose which logger to use :|

Reply:
I'm not sure, is this a suggestion for a change that I can make here, or does this mean "we need a followup to rework logging in rehearsals"?

Reply:
The latter.